### PR TITLE
Added  details link tap event handling.

### PR DESCRIPTION
**Pull request by marcinkurtz**
Extended RecordingConsentView with details link tap recognition handling.

### CODE REVIEW

**Review by DorotaMacura — APPROVED** [review]

File: Xamarin.UserConsent/UI/RecordingConsentView.xaml.cs [hunk 1]
```diff
@@ -126,5 +128,10 @@ private void SetLabelColors()
                 YesLabel.TextColor = Colors.DisabledTextColor;
             }
         }
+
+        private void OnDetailsLinkTaped(object sender, EventArgs e)
```
Comment (mgawron-fp) [inline Xamarin.UserConsent/UI/RecordingConsentView.xaml.cs]: Taped -> Tapped

Comment (marcinkurtz) [reply Xamarin.UserConsent/UI/RecordingConsentView.xaml.cs]: Fixed.

**Review by rnowosielski — APPROVED** [review]

**Review by michaldepta — COMMENTED** [review]
A new version won't be published unless you increment version number.

**Comment by marcinkurtz** [comment]
@michaldepta I waiting for Dorota changes regarding styling to do a one time package release - single version.

**Pull request merged**